03-using-phpstorm.md

```diff
@@ -1,11 +1,28 @@
 Using PHPStorm
 ==============
 
-We have collected some useful tips for working with PHPStorm on our projects
+We have collected some useful tips for working with PHPStorm on our projects.
 
 
-## PHP-CS-Fixer
+## Using the proper Code Style settings
 
+PHPStorm allows you to define some advanced formatting options for several languages,
+and then apply them with a single hotkey to the file you are currently editing.
+These options can be changed, saved and then exported for use by others.
+To keep a consistent codestyle in our projects, you are advised to use the style shown below:
+
+1. Copy the file from [here](assets/codestyles/TreeHouse.xml) and save it under ``~/Library/Preferences/WebIde70/codestyles/TreeHouse.xml``
```

Comment:
Bij mij is dit WebIde80. Dit nummer scheelt per versie, misschien daar nog iets over opnemen als sidenote?